### PR TITLE
Replace PNG images with SVG icons

### PR DESCRIPTION
**Pull request by arnautov-anton**

### CODE REVIEW

File: src/components/RepostButton.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ import ReactionToggleIcon from './ReactionToggleIcon';
 
 import { StreamApp, withTranslationContext } from '../Context';
 
-import repostButtonActive from '../images/repost-active@1x.png';
-import repostButtonInactive from '../images/repost@1x.png';
+// import repostButtonActive from '../images/repost-active@1x.png';
+// import repostButtonInactive from '../images/repost@1x.png';
+
```
Comment (mahboubii) [inline src/components/RepostButton.js:8]:
```suggestion
```

**Review by mahboubii — CHANGES_REQUESTED** [review]
placeholder.png file also should be replaced

File: src/assets/icons.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {...props}
+    //  class="svg-inline--fa fa-thumbs-up fa-w-16"
+    className="raf-reaction-icon__image"
+    data-icon="thumbs-up"
+    role="img"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 512 512"
+  >
+    <path
+      fill="currentColor"
```
[the remark this answers is not in the document]
Comment (arnautov-anton) [reply src/assets/icons.tsx:21]: CanIUse [says](https://caniuse.com/?search=currentColor) it is. :)

[the remark this answers is not in the document]
Comment (mahboubii) [reply src/assets/icons.tsx]: nice

File: src/components/icons.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+export const ThumbsUpIcon = (props: Icon): JSX.Element => (
+  <svg
+    {...props}
+    className="raf-reaction-icon__image"
+    data-icon="thumbs-up"
+    role="img"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 512 512"
```
Comment (mahboubii) [inline src/components/icons.tsx:17]: same for other Icons to allow overriding the default  attributes
```suggestion
    className="raf-reaction-icon__image"
    data-icon="thumbs-up"
    role="img"
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 512 512"
     {...props}
```

File: src/components/icons.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React, { SVGProps } from 'react';
+
+type Icon = SVGProps<SVGSVGElement>;
```
Comment (mahboubii) [inline src/components/icons.tsx:3]:
```suggestion
type IconProps = SVGProps<SVGSVGElement>;
```

File: src/components/icons.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+export enum Color {
+  Active = '#0ba8e0',
+  Inactive = '#7a8287',
+}
+
+export const ThumbsUpIcon = (props: Icon): JSX.Element => (
```
Comment (mahboubii) [inline src/components/icons.tsx:10]: no need to specefically set the return type
```suggestion
export const ThumbsUpIcon = (props: IconProps) => (
```

**Pull request merged**